benchmarks/benchmark-join2.js

```diff
@@ -12,25 +12,25 @@ let ipfs
 let log1, log2
 
 // Metrics
-let totalQueries = 0
-const queryLoop = async () => {
-  try {
-    await Promise.all([
-      log1.append('a' + totalQueries),
-      log2.append('b' + totalQueries)
-    ])
+// const totalQueries = 0
```

Comment:
Not sure if this was being used since it didn't seem to be anywhere. I can uncomment if need be in a future PR.